action.yml

```diff
@@ -13,7 +13,7 @@ runs:
       run: |
         status=0
         dirs=$(find . \( -name vendor -o -name '[._].*' -o -name node_modules \) -prune -o -name go.mod -print | sed 's:/go.mod$::')
-        len=${#dirs[@]}
+        len=$(echo "$dirs" | wc -l | tr -d ' ')
```

Comment:
why tr? I'm pretty sure it's unnecessary:

```
len=$(echo foo bar | wc -l); echo $len
```

Reply:
Err, I meant:

```
$ len=$((echo foo; echo bar) | wc -l); echo $len
2
```

Reply:
On OSX, `wc -l` adds a bunch of useless spaces:
```bash
❯ cat README.md | wc -l
      37
```

Reply:
And with the additional spaces bash doesn't interpret it as an integer any more: `if [[ $len > 1 ]]; then` would fail.

Reply:
SGTM